PlanningBiblio/PlanningBiblio
# MT44184: use symfony/html-sanitizer instead of htmlpurifier

Test plan:
    
     1) Edit a planning comment and check that the input in sanitized.
     2) Save a planning as model and check that the model name is sanitized.
     3) prove tests/Include/IncludeSanitizeTest.php
    
    Note:
     htmlpurifier used to html-encode PHP code, html-sanitizer removes it.
